packages/demos-babylon/README.md

```diff
@@ -2,18 +2,93 @@
 
 This directory contains a number of demonstrations, each focused on a different feature of the Sumerian Hosts API.
 
-## Demo setup
+### AWS Setup
```

Comment:
nit: should it be level 2 heading or 3?

Reply:
Thanks for catching. It definitely looks like my heading levels are off. I will fix in the next CR update.